src/build/env.py

```diff
@@ -68,6 +72,15 @@ def _should_use_virtualenv() -> bool:
     )
 
 
+def _subprocess(cmd: List[str]) -> None:
+    """Invoke subprocess and output stdout and stderr if it fails."""
+    try:
+        subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+    except Exception as e:  # pragma: no cover
+        print(e.output.decode(), end='')  # type: ignore  # e.output isn't type hinted but is documented
```

Comment:
perhaps open an issue to typeshed and link it here instead?

Reply:
Like pointed out above, I am catching `Exception`, which will obviously not have `e.output` :sweat_smile: